foursight_core/fs_connection.py

```diff
@@ -46,18 +46,20 @@ def __init__(self, fs_environ, fs_environ_info, test=False, use_es=True, host=No
         self.ff_env = fs_environ_info['ff_env']
         self.ff_es = fs_environ_info['es']
         self.ff_bucket = fs_environ_info['bucket']
+        self.redis = None
+        self.redis_url = None
         try:
             if 'redis' in fs_environ_info:
-                self.redis = RedisBase(create_redis_client(url=fs_environ_info['redis']))
+                self.redis_url = fs_environ_info['redis']
+                self.redis = RedisBase(create_redis_client(url=self.redis_url))
             elif 'REDIS_HOST' in os.environ:  # temporary patch in until env config is fully sorted - Will
-                self.redis = RedisBase(create_redis_client(url=os.environ['REDIS_HOST']))
-            else:
-                self.redis = None
+                self.redis_url = os.environ['REDIS_HOST']
+                self.redis = RedisBase(create_redis_client(url=self.redis_url))
         except redis.exceptions.ConnectionError:
             PRINT('Cannot connect to Redis')
             PRINT('This error is expected when deploying with remote (ElastiCache) Redis')
         PRINT(self.redis)
-        PRINT(os.environ.get('REDIS_HOST', 'no-redis-host'))
+        PRINT(self.redis_url)
```

Comment:
These should say what they are printing. Something like (untested):
```
PRINT(f"self.redis_url={self.redis_url}")
PRINT(f"self.redis={self.redis}")
```
Note that I also printed these in the reverse order for two reasons:
* The redis URL is likely simpler data, so if there's a bug in how self.redis prints, you'll have some partial info to help debug it.
* This is the order in which the variable are initialized in code.

Reply:
Gotcha, thanks.